### PR TITLE
use full path beside absolute path

### PR DESCRIPTION
**Pull request by shayanypn**

### CODE REVIEW

**Comment by moshfeu** [comment]
You mean for people who getting this from Github?

**Comment by shayanypn** [comment]
@moshfeu  for those who want to check it from the blog, it redirect to `https://blog.codingcoach.io/mock-openings/`. of course its working, but not good idea to have it with subdomain of `blog`

File: content/blog/mock-openings-practice-interviewing/index.md [hunk 1]
```diff
@@ -28,4 +28,4 @@ For make it clear, we have some steps in this process:
 Your information, interviewing process, and your evaluation keep private with us.
 
 
-For starting, visit link [Mock Opening](/mock-openings/).
+For starting, visit link [Mock Opening](https://codingcoach.io/mock-openings).
```
Comment (brentmclark) [inline content/blog/mock-openings-practice-interviewing/index.md:31]: This link results in a 404 for me, but it seems to work with the trailing slash.  Try this instead?

```suggestion
For starting, visit link [Mock Opening](https://codingcoach.io/mock-openings/).
```

**Comment by moshfeu** [comment]
@crysfel don't we shut down this subdomain?

**Comment by crysfel** [comment]
Apparently we didn't :( but I just did! No more https://blog.codingcoach.io/ to avoid any confusion :)

**Comment by crysfel** [comment]
Closing this PR

**Pull request closed without merging**